### PR TITLE
Apache Tez initialization action

### PR DESCRIPTION
**Pull request by evilsoapbox**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,6 +18,10 @@ This repository presently offers the following actions for use with Cloud Datapr
 * Install packages/software on the cluster
   * [Apache Kafka](http://kafka.apache.org)
   * [Apache Oozie](http://oozie.apache.org)
+<<<<<<< HEAD
```
Comment (dennishuo) [inline README.md]: Looks like a remaining merge conflict.

**Comment by evilsoapbox** [comment]
GitHub-fu weak. Sigh. Fixed. :)

**Comment by Spark2Ignite** [comment]
Still can't see it in GitHub's repo root...

**Comment by dennishuo** [comment]
Merged with https://github.com/GoogleCloudPlatform/dataproc-initialization-actions/commit/eccde8ee9f8dbec8b3c8d567343bad4992a99d19

**Comment by Spark2Ignite** [comment]
Thank you!

**Pull request closed without merging**